Automattic/woocommerce-payments
# Refactor WC_Payment_Gateway_WCPay Part 2

#### Changes proposed in this Pull Request
Related to #823 
This PR is the second part of a series of refactors of the WC_Payment_Gateway_WCPay class.

#### Testing instructions
- As a shopper add a product to the cart
- Make sure you can see the card inputs
- Buy the product
- Test again using UPE
- As a shopper buy any product using WooPay

- Change and save WC Pay settings via wp-admin and make sure they are saved correctly and reflect in the shopper flow

- Execute other [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) that you think are necessary


-------------------


**Post merge**
